src/App.tsx

```diff
@@ -1,24 +1,214 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Select } from './types/Select';
+import {
+  getTodos,
+  deleteTodo,
+  patchTodoCompleted,
+  patchTodoTitle,
+} from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Errors } from './types/Errors';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+export const USER_ID = 9960;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
```

Comment:
It's better to use empty array
```suggestion
  const [todoList, setTodoList] = useState<Todo[]>([]);
```